app/presenters/national_applicability.rb

```diff
@@ -23,8 +23,8 @@ def applies_to
 
 private
 
-  def translated_format_name(count)
-    I18n.t("content_item.format.#{format}", count: count).downcase
+  def translated_schema_name(count)
+    I18n.t("content_item.format.#{schema_name}", count: count).downcase
```

Comment:
Fair enough. It might be worth leaving a comment in the code so that future intend is not forgotten.